### DISCUSSION TITLE
Frontend Mapping Libraries

### DISCUSSION THREAD

**Discussion by eigenbeam** [nsidc/tech-radar#30] [Languages and Frameworks]
In past (inactive) projects we've used both [Leaflet](https://leafletjs.com/) and [OpenLayers](https://openlayers.org/) for map interfaces, (re-)projection, and related geospatial visualization & operations. Recently, the general consensus has been to prefer OpenLayers over Leaflet; the main reason (as I recall) was better support in OL for polar projections. That may no longer be true, and the libraries may be more similar. 

I'd like to discuss OpenLayers, Leaflet, and other similar JS libraries--what are their pros & cons, what are they good & not so good for, etc.

**Comment by bruceraup** [comment]
GLIMS Glacier Viewer is (happily) using OpenLayers 3.
